tutorial/README.md

```diff
@@ -13,31 +13,41 @@ These labs provide an insight on the advantages of using Helm over using Kuberne
+
+The [Helm 2 Support Plan](https://helm.sh/blog/2019-10-22-helm-2150-released/#helm-2-support-plan) documents a 1 year "maintenance mode" for Helm v2. It states the following:
+- 6 month bug fixes until May 13 2020
+- 6 month security fixes until November 13 2020
+- At 1 year on November 13 2020, support for Helm v2 will end
+
+***Note: This tutorial uses Helm v3. If you are using Helm v2 then go to the [helm-v2](https://github.com/IBM/helm101/tree/helm-v2/tutorial) branch.***
+
 # Prerequisites
 
 * Have a running Kubernetes cluster. See the [IBM Cloud Kubernetes Service](https://cloud.ibm.com/docs/containers/cs_tutorials.html#cs_cluster_tutorial) or [Kubernetes Getting Started Guide](https://kubernetes.io/docs/setup/) for details about creating a cluster.
-* Have Helm installed and initialized with the Kubernetes cluster. See [Installing Helm on IBM Cloud Kubernetes Service](Lab0/README.md) or the [Helm Quickstart Guide](https://v2.helm.sh/docs/using_helm/#quickstart) for getting started with Helm.
+* Have Helm installed and initialized with the Kubernetes cluster. See [Installing Helm on IBM Cloud Kubernetes Service](Lab0/README.md) or the [Helm Quickstart Guide](https://helm.sh/docs/intro/quickstart/) for getting started with Helm.
 
 # Helm Overview
 
 Helm is a tool that streamlines installation and management of Kubernetes applications. It uses a packaging format called "charts", which are a collection of files that describe Kubernetes resources. It can run anywhere (laptop, CI/CD, etc.) and is available for various operating systems, like OSX, Linux and Windows.
 
 ![helm-architecture](images/helm-architecture.png)
 
-It has a client-server architecture with the client called `helm` and the server called `Tiller`. The client is a CLI which users interact with to perform different operations like install/upgrade/delete etc. The client interacts with Tiller and the chart repository. Tiller interacts with the Kubernetes API server. It renders Helm template files into Kubernetes manifest files which it uses to perform operations on the Kubernetes cluster via the Kubernetes API. See the [Helm Architecture](https://v2.helm.sh/docs/architecture/) for more details. 
+It has a client architecture with the client called `helm` and a Go library which encapsulates the Helm logic so that it can be leveraged by different clients. The client is a CLI which users interact with to perform different operations like install/upgrade/delete etc. The client interacts with the Kubernetes API server. It renders Helm template files into Kubernetes manifest files which it uses to perform operations on the Kubernetes cluster via the Kubernetes API. See the [Helm Architecture](https://helm.sh/docs/topics/architecture/) for more details. 
```

Comment:
What are the different clients here?

Reply:
No supported clients, its just that the SDK allows users to plug in their own client. The only client supported by helm is `helm`.